scripts/start.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 npm run build-dev &
-AWS_PROFILE=composer sbt run
+AWS_PROFILE=composer sbt run -jvm-debug 5005
```

Comment:
Did you mean to leave this here?

Reply:
Personally don't see the harm it does especially since `scripts/start.sh` is used just for development. It's convenient for the debugger to be available. :)